.github/workflows/build_docs.yml

```diff
@@ -12,20 +12,50 @@ on:
       - 'docs/**'
       - '.github/workflows/build_docs.yml'
   workflow_dispatch:
+  # Build docs if Animate, Goalie, Movement, or UM2N have been updated in the last 24h
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I gather this means every night at midnight? If so, please mention this time because the cron syntax is far from intuitive.

Reply:
Agreed :) done in a7b0453b6d40df17fe41ea09d4f7489991fe133a